### PR TITLE
docs: create basic story for < ComponentDescription />, < UnstyledList /> + < UnstyledListItem />

### LINKED ISSUE #89: docs: Create basic story for < ComponentDescription />, < UnstyledList /> + < UnstyledListItem />
[Monday ticket](https://monday.monday.com/boards/3532714909/pulses/5350282741)

Add basic story by structure similar to existing ones
Use `UnstyledListItem` in `UnstyledList` story

- https://github.com/mondaycom/vibe-storybook-components/blob/master/CONTRIBUTING.md
- https://github.com/mondaycom/vibe-storybook-components/blob/master/COMPONENTS_DOCUMENTATION_GUIDELINES.md

### PR DESCRIPTION
**Pull request by Dhoni77**
Closes #89

- Added stories for < ComponentDescription />, < UnstyledList /> + < UnstyledListItem />

### CODE REVIEW

File: src/components/unstyled-list/__stories__/unstyled-list.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import * as UnstyledListStory from './unstyled-list.stories';
+
+<Meta of={UnstyledListStory} />
+
+# UnstyledList
+
+## Overview
+
+UnstyledList is a component that displays the items in an unordered list.
+This component can be used as a replacement for `ul`.
```
Comment (SergeyRoyt) [inline src/components/unstyled-list/__stories__/unstyled-list.mdx:11]:
```suggestion
This component can be used as a replacement for `ul` and it's child component `UnstyledListItem` can be used as a replacement for `li`.
```

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
One small note, the rest is great!

**Comment by Dhoni77** [comment]
cc @SergeyRoyt

**Review by SergeyRoyt — APPROVED** [review]
Great! Thank you!! 🙏

**Pull request merged**